src/Illuminate/Collections/LazyCollection.php

```diff
@@ -808,6 +808,28 @@ public function only($keys)
         });
     }
 
+    /**
+     * Get the items that are not filled.
+     *
+     * @param  mixed  $keys
+     * @return static
+     */
+    public function blank()
+    {
+        return $this->passthru('blank', func_get_args());
```

Comment:
I'd recommend to use `LazyCollection::filter()` that yields the filtered values straightaway instead of calling `passthru()` which turns the iterator to array before moving back to the lazy collection:
```suggestion
        return $this->filter('blank');
```

If eventually the methods `blank()` and `filled()` are identical for both collections and lazy collections, we may move those methods in the `EnumeratesValues` trait